benefits/eligibility/forms.py

```diff
@@ -18,9 +18,9 @@ class EligibilityVerifierSelectionForm(forms.Form):
     action_url = "eligibility:index"
     method = "POST"
 
-    verifier = forms.ChoiceField(label="", widget=widgets.RadioSelect)
+    verifier = forms.ChoiceField(label=_("eligibility.pages.index.label"), widget=widgets.RadioSelect)
```

Comment:
I noticed an asterisk by this label... seems like the CA Web Template is adding it (?)

Should we try to remove it since it's not in the Figma design?

![image](https://user-images.githubusercontent.com/25497886/192619340-feefe1d6-b438-4da7-8c46-f41a16c43f5a.png)

Reply:
Yeah I think we will need to remove it with CSS later. 

It's a little confusing actually:
- On this page, the radio button question is required, and it does not have the required asterisks.
- On the next page, both the form input fields are required, and they DO have the required asterisks. 

@srhhnry

Reply:
Oh wait, we're the ones adding the asterisk! https://github.com/cal-itp/benefits/blob/dev/benefits/core/templates/core/includes/form.html#L19

Reply:
So... I am thinking we should remove this line `{% if field.field.required %}<span class="required-label">*</span>{% endif %}` from the template entirely and then manually add an asterisks to the form page labels. What do you think? I am not sure if having the * be in a <span> is something that is required for a11y/screenreaders or something.

Reply:
By "form page labels" for this page, do you mean the radio button labels?

Reply:
To this page -- add these `*` manually as copy (rather than `{% if field.field.required %}<span class="required-label">*</span>{% endif %}`)
<img width="547" alt="image" src="https://user-images.githubusercontent.com/3673236/192634088-ea442a0e-3205-4d6f-ba6f-baff8fffebc7.png">

Reply:
Hmm, I'd rather not have to manually put the `*` into copy... the current behavior seems more correct (asterisk reflects the actual markup). 

It sounds like what you're getting at is that we might not always want an asterisk to show even though the field is required. Maybe we could add a property to the form or something that's like `show_asterisk_for_required`?

Reply:
We will make a new ticket regarding the * once we get an answer from Design as to what to do. In the meantime, going to resolve conflicts and get this PR re-reviewed for merging.

Reply:
We decided to leave the * in all required fields (so, all fields) for now. Next sprint will focus more on form elements/styling.